src/CozyUtils.js

```diff
@@ -1,17 +1,18 @@
-const { log } = require('cozy-konnector-libs')
+const { log, cozyClient } = require('cozy-konnector-libs')
+const { Q } = require('cozy-client')
 const get = require('lodash/get')
-const initCozyClient = require('./helpers/initCozyClient')
```

Comment:
If this file is not imported anymore anywhere else, you can delete it.

Reply:
done :white_check_mark: